packages/plugin-kit/src/createSliceMachineHelpers.ts

```diff
@@ -74,6 +74,28 @@ export class SliceMachineHelpers {
 		};
 	};
 
+	updateProjectConfig = async (
+		config: SliceMachineConfig,
```

Comment:
But we don't have any object properties apart from a potential adapter config? So everything should be overridden? Gave it a simple go, let me know.

Reply:
Eventually decided against that for now:
- There's no clear/convenient way of merging the config correctly as you might want to remove a value, add one in an array, filter them.
- Because of the above, you still have to read the project on the plugin's end to decide on what patches you want to make.

Reply:
Makes sense to me! Using the current implementation still lets someone use a lib like defu internally (though that would be unusual).